packages/graph-explorer/src/utils/logger.ts

```diff
@@ -14,16 +14,28 @@ at runtime.
 */
 
 export default {
+  /** Calls `console.log` if the app is running in DEV mode. */
+  debug(message?: any, ...optionalParams: any[]) {
+    env.DEV && optionalParams.length > 0
```

Comment:
This expression will always write to the console. Please fix comment or code.

Reply:
Dang. Good catch. My understanding of how JS will evaluate this line is totally wrong.